src/main/java/org/fundacionjala/coding/dennis/FizzBuzz.java

```diff
@@ -0,0 +1,54 @@
+package org.fundacionjala.coding.dennis;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ *
+ */
+public  final class FizzBuzz {
+
+    public static final int THREE = 3;
+    public static final int FIVE = 5;
+    public static final int MCM = THREE * FIVE;
+
+    /**
+     *
+     */
+    private FizzBuzz() {
+    }
+
+    /**
+     *
+     * @param number numero a ser evaluado.
+     * @return cadena resultado.
+     */
+    public static String singleFizzBuzz(int number) {
+        String stringResultFizzBuzz = Integer.toString(number);
+        CharSequence three = "3";
+        CharSequence five = "5";
+        if (number % MCM == 0) {
+            return "FizzBuzz";
+        }
+        if (number % THREE == 0 || stringResultFizzBuzz.contains(three)) {
+            return "Fizz";
+        }
+        if (number % FIVE == 0 || stringResultFizzBuzz.contains(five)) {
+            return "Buzz";
+        }
+        return stringResultFizzBuzz;
+    }
+
+    /**
+     *
+     * @param limit cantidad maxima de elementos en lista.
+     * @return Lista de los elementos devueltos.
+     */
+    public static ArrayList<String> listFizzBuzz(int limit) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The return type of this method should be an interface such as "List" rather than the implementation "ArrayList". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1319)